pkg/cluster/internal/providers/docker/util.go

```diff
@@ -85,3 +86,34 @@ func mountDevMapper() bool {
 
 	return storage == "btrfs" || storage == "zfs" || storage == "xfs"
 }
+
+// rootless: use fuse-overlayfs by default
+// https://github.com/kubernetes-sigs/kind/issues/2275
+func mountFuse() bool {
+	cmd := exec.Command("docker", "info", "--format", "{{json .}}")
```

Comment:
should we just wrap the info method like we did with podman?

Reply:
what do you mean exactly? I see both the same :thinking:

Reply:
I mean one of them actually re-uses the `info` method without reimplementing the contents

Reply:
I've removed those dependencies on this PR in both places, but I get your point now ... since we depend on the rootless variable of info() seems good to add that dependency

Reply:
updated